src/pages/RoomMembersPage.tsx

```diff
@@ -174,7 +174,7 @@ function RoomMembersPage({report, session, policies}: RoomMembersPageProps) {
         participants.forEach((accountID) => {
             const details = personalDetails[accountID];
 
-            if (!details) {
+            if (!details || (details.isOptimisticPersonalDetail && participants.some((accID) => accID !== accountID && details.login === personalDetails[accID]?.login))) {
```

Comment:
NAB. Can you please declare a variable for this condition
```javascript
details.isOptimisticPersonalDetail && participants.some((accID) => accID !== accountID && details.login === personalDetails[accID]?.login)
```

And add a comment to explain why we need it?